utility/wiringPi/RF24_arch_config.h

```diff
@@ -12,13 +12,13 @@
 #define RF24_LINUX
 
 #include <stddef.h>
-#include "spi.h"
-#include "wiringPi.h"
 #include <stdint.h>
 #include <stdio.h>
 #include <time.h>
 #include <string.h>
 #include <sys/time.h>
```

Comment:
again with the 2 time.h headers...

Reply:
It seems I didn't review the wiringPi driver code for unnecessary `#include`s.

This lead me to review wiringPi/spi.cpp which seems to only use "channel" 0 (hard-coded as `RF24_SPI_CHANNEL`). The wiringPi docs are somewhat missing.

I'm guessing the "channel" refers to the CSN pin for bus 0 since the older RPi1 had limited GPIO offerings. If "channel" does refer to the CSN pin, then why not use the `SPI::begin()` param `csn_pin` (as `channel = csn_pin % 10`)?

I've (locally) already replaced the `printf(); abort()` calls with `throw SPIException()`. I think most of the `printf()` calls were just debug statements that never got cleaned up before release.

Reply:
I remove unnecessary includes for time.h here; they were not used.

I also updated wiringPi wrapper's `SPI::begin()` to allow for CE0 and CE1 "channels". And, I pushed the switch to `SPIException` with more informative prompts.